js/script.js

```diff
@@ -1,74 +1,88 @@
 /* eslint-disable no-unused-vars */
+/* eslint-disable class-methods-use-this */
 
-const booksElement = document.querySelector('.books');
+// const booksElement = document.querySelector('.books');
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.

Reply:
Resolved.